static/src/js/components/CollectionDetails/CollectionDetailsFeatureGroup.js

```diff
@@ -2,7 +2,7 @@ import { createLayerComponent } from '@react-leaflet/core'
 
 import { buildLayer } from '../../util/map/layers'
 
-const createLayer = (props, context) => {
+export const createLayer = (props, context) => {
```

Comment:
I made this more accessible just to test it. In the  ```
cy.get('.collection-details-minimap').within(() => {
        cy.get('.leaflet-interactive').should('have.attr', 'd', 'M0 180L360 180L360 0L0 0L0 180z')
      }) 
```
we are looking at the mini-map as well

Reply:
But the named export isn't ever used within the code, so the test isn't doing a great job of replicating conditions. Just test it in e2e tests and don't worry about adding it to jest